src/coredump/coredump.c

```diff
@@ -756,7 +756,6 @@ static int process_socket(int fd) {
 
                 iovec[n_iovec].iov_len = l;
                 iovec[n_iovec].iov_base = malloc(l + 1);
-
```

Comment:
nope, this was intended actually, but i snuck it into the commit, even though it is unrelated... was too trivial to add a seperate commit i thought.

Reason: we generally try to keep function calls and the follow-up error checking together. and in this case we hence should check the return value of malloc() without a whitespace line in between